includes/FormFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -86,6 +86,7 @@ private static function buildDescriptorCore(
 					'type' => 'check',
 					'label-message' => "managewiki-label-{$mwAction}wiki",
 					'default' => false,
+					'disabled' => $wiki->getDBname() == $config->get( 'CreateWikiGlobalWiki' ),
```

Comment:
I think maybe we should just add this check to the above if statement, as we also don't want to lock it either, so that way (above) we can just hide this if on the central wiki itself, to act like any other wikis for that one.

Reply:
I've moved the check to the if statement